pilot/pkg/model/validation_test.go

```diff
@@ -2606,6 +2606,30 @@ func TestValidateVirtualService(t *testing.T) {
 				}},
 			}},
 		}, valid: false},
+		{name: "multi short names", in: &networking.VirtualService{
+			Hosts: []string{"svc1", "svc2"},
+			Http: []*networking.HTTPRoute{{
+				Route: []*networking.HTTPRouteDestination{{
+					Destination: &networking.Destination{Host: "foo.baz"},
+				}},
+			}},
+		}, valid: false},
+		{name: "multi services FQDN", in: &networking.VirtualService{
+			Hosts: []string{"svc1.default.svc.cluster.local", "svc2.default.svc.cluster.local"},
+			Http: []*networking.HTTPRoute{{
+				Route: []*networking.HTTPRouteDestination{{
+					Destination: &networking.Destination{Host: "foo.baz"},
+				}},
+			}},
+		}, valid: false},
+		{name: "both service shotname and service FQDN specified", in: &networking.VirtualService{
+			Hosts: []string{"svc1", "svc2.default.svc.cluster.local"},
+			Http: []*networking.HTTPRoute{{
+				Route: []*networking.HTTPRouteDestination{{
+					Destination: &networking.Destination{Host: "foo.baz"},
+				}},
+			}},
+		}, valid: false},
```

Comment:
I don't understand. When is it valid to have multiple hosts? It seems like now you are saying it is never ok to have multiple hosts

Reply:
Sorry, it makes you confused, i will add multi hosts cases.